lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -153,6 +161,20 @@ spec = do
         r2 <- joinStakePool ctx p2 (w, "Secure Passprase")
         expectResponseCode HTTP.status501 r2
 
+    it "STAKE_POOLS_JOIN_01 - Cannot join non-existant stakepool" $ \ctx -> do
+        let pId =
+                "4f8d686a02c6e625b5a59cc9e234f32e5d72987012f9c25c9a6b60ddade197d9" :: Text
```

Comment:
I know that I need to use `pIdAbsent` here in the end. I just get here 
```
\"4f8d686a02c6e625b5a59cc9e234f32e5d72987012f9c25c9a6b60ddade197d9\"
```
when using `show pIdAbsent`

Reply:
Alternatively, use `0000000.....00000` for an unknown stake pool. Chances of getting a stake pool with such id are null (1 in 2^32 exactly)

Reply:
used 00000....00000 and refactored the test case a little bit